versions/3.0.md

```diff
@@ -1080,21 +1080,21 @@ content:
 ```
 
 #### <a name="contentTypeObject"></a>Content Type Object
-Each content type object provides schema and examples for a the media type identified by its key.  Content Type objects can be used in a [content object](#contentObject).   
+Each Content Type Object provides schema and examples for the media type identified by its key.  Content Type Objects can be used in a [Content Object](#contentObject).   
 
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
 <a name="contentTypeSchema"></a>schema | [Schema Object](#schemaObject) | The schema defining the type used for the request body.
-<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the content type.  Each example in the Examples array SHOULD be in the correct format as specified in the _content_ type.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
-<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object SHOULD be in the correct format as specified in the _content_ type.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value SHALL _override_ the the example provided by the schema.
+<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the Content-Type.  Each example in the Examples array SHOULD be in the correct format as specified in the media type.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
```

Comment:
I don't really like the use of `Content-Type` here as it makes me think that it is an example of the value in a `Content-Type` header.